plugins/email-messages/impl/pom.xml

```diff
@@ -49,5 +49,11 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+      <dependency>
```

Comment:
This won't work in our build pipeline.  pentaho-platform extensions gets built here: https://github.com/pentaho/jenkins-pipelines/blob/10.3.0.0/resources/builders/suite-release.yaml#L329

This plugin gets built as part of the kettle lowdeps here, which is earlier: https://github.com/pentaho/jenkins-pipelines/blob/10.3.0.0/resources/builders/suite-release.yaml#L209

Either this dependency needs to be removed or this plugin needs to be moved to the highdeps section.